hw/dv/doc/dv_doc_template.md

```diff
@@ -115,6 +115,6 @@ $ cd hw/ip/foo/dv
 $ make TEST_NAME=foo_smoke
 ```
 
-## Testplan
-<!-- TODO: uncomment the line below after adding the testplan -->
+## testplan
```

Comment:
```suggestion
## DV plan
```

Reply:
@sriyerg 
I think this should be
'## Testplan'
as the pointer points to the testplan not the dv plan.
at least if it is still the plan to have a different hjson file for the functional coverage?

Reply:
fixed